api/tests/e2e/service_instances_test.go

```diff
@@ -0,0 +1,74 @@
+package e2e_test
+
+import (
+	"net/http"
+
+	"github.com/go-resty/resty/v2"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
+	rbacv1 "k8s.io/api/rbac/v1"
+)
+
+var _ = Describe("Service Instances", func() {
```

Comment:
As this PR introduces a `get service` endpoint, there should be an e2e test for it as well (provided that we need `Get` at all, see my previous comments)

Reply:
We are only introducing the ability to get a service instance on the repo level. There is no endpoint for getting the service instance on the handler level, so it is not possible to add an e2e test for this.

Reply:
I don't think we'll be adding that endpoint soon (`GET /v3/service_instances/:guid`) since the CLI is always fetching these via list calls (filtered by `name` and `space_guid`) -- at least for the CLI commands we're targeting right now.